docker/utils/utils.py

```diff
@@ -952,6 +953,8 @@ def format_env(key, value):
         return '{key}={value}'.format(key=key, value=value)
     return [format_env(*var) for var in six.iteritems(environment)]
 
+def duration_from_string(duration_string):
+    return timeparse(duration_string) * 1000000000
```

Comment:
I don't think we should parse duration strings ourselves and introduce another dependency. Users of docker-py can instead use whatever library they prefer, whether it's `timeparse` or `arrow` or doing the arithmetic themselves.